BraveWallet/Extensions/Text+URLOrigin.swift

```diff
@@ -19,7 +19,8 @@ extension Text {
         let originStart = origin[origin.startIndex..<range.lowerBound]
         let etldPlusOne = origin[range.lowerBound..<range.upperBound]
         let originEnd = origin[range.upperBound...]
-        self = Text(originStart) + Text(etldPlusOne).bold() + Text(originEnd)
+        let zwspEtldPlusOne = "\u{200b}\(String(etldPlusOne))"
```

Comment:
What does VoiceOver read this out as? Do we need to set `accessibilityLabel` somewhere?